src/index.scss

```diff
@@ -479,7 +487,7 @@ header {
 }
 
 .font-style {
-  font-family: "Inter";
+  font-family: inherit;
```

Comment:
This change should also be reverted as it will be resolved once the theme is applied.

Reply:
This issue has been resolved and merged. Please rebase this branch.

Reply:
Fixed